process/block/preprocess/transactions.go

```diff
@@ -1076,6 +1074,7 @@ func (txs *transactions) ProcessMiniBlock(miniBlock *block.MiniBlock, haveTime f
 			return processedTxHashes, err
 		}
 
+		processedTxHashes = append(processedTxHashes, miniBlockTxHashes[index])
```

Comment:
Rename `index` to `txHash` (optional).

Reply:
it is actually the index. that is a list of hashes.